src/main/java/com/aws/greengrass/device/session/SessionManager.java

```diff
@@ -77,6 +87,15 @@ public void closeSession(String sessionId) {
         closeSessionInternal(sessionId);
     }
 
+    /**
+     * Updates Session Configuration.
+     *
+     * @param sessionConfig desired session configuration
+     */
+    public void updateSessionConfig(SessionConfig sessionConfig) {
```

Comment:
this isn't really updating, it is setting.

Reply:
updated